README.md

```diff
@@ -31,19 +31,25 @@ Progress: https://github.com/ipfs/community/discussions/573
   - [x] Render empty directories
   - [x] Resolve `index.html` in a path
 - `mutable.html` Tests (experimental, some things subject to change)
-  - [x] JS `fetch('ipfs://CID/example.txt', {method: 'POST'})`
-  - [x] JS `fetch('ipns://CID/', {method: 'POST'})`
-  - [x] JS `fetch('ipns://CID/example.txt', {method: 'POST'})`
-  - [x] JS `fetch('ipfs://CID/', {method: 'POST', body: new FormData})`
-  - [x] JS `fetch('ipfs://CID/example.txt', {method: 'DELETE'})`
-  - [x] JS `fetch('ipns://CID/example.txt', {method: 'DELETE'})`
+  - [x] JS `fetch('ipfs://<CID>/example.txt', {method: 'PUT'})`
+  - [x] JS `fetch('ipfs://<CID>/', {method: 'PUT', body: new FormData})`
+  - [x] JS `fetch('ipfs://<CID>/example.txt', {method: 'DELETE'})`
+  - [x] JS `fetch('ipns://<key>', {method: 'POST', body: <CID>})`
+  - [x] JS `fetch('ipns://<key>/example.txt', {method: 'POST', body: <CID>})`
```

Comment:
:thought_balloon: sending CID in the body feels weird, but I can't put my finger on why :sweat_smile: 

I was thinking that maybe it is squatting too much, what if one wants to do IPNS in userland, maintain keys on their own, create, sign record in JS, and then send it to `ipns://` only for publishing? I believe we should make sure a primitive for doing it exists.

I guess that could be represented as `POST` with `Content-Type: application/vnd.ipfs.ipns.record` ?